src/callbacks/alive.jl

```diff
@@ -32,9 +32,22 @@ function Base.show(io::IO, cb::DiscreteCallback{Condition,Affect!}) where {Condi
   alive_callback = cb.affect!
   print(io, "AliveCallback(alive_interval=", alive_callback.alive_interval, ")")
 end
-# TODO: Taal bikeshedding, implement a method with more information and the signature
-# function Base.show(io::IO, ::MIME"text/plain", cb::DiscreteCallback{Condition,Affect!}) where {Condition, Affect!<:AliveCallback}
-# end
+
+function Base.show(io::IO, ::MIME"text/plain", cb::DiscreteCallback{Condition,Affect!}) where {Condition, Affect!<:AliveCallback}
+  if get(io, :summary, false)
```

Comment:
That's a bit confusing for me. If I `get(io, :summary) == true`, I would expect to get a summary, i.e., a shorter output. Hence, I would prefer to use `!get(io, :compact, true)` here (as mentioned above).

Reply:
Yep, I can fully understand it. Let's keep this discussion in the other thread, though.